src/components/Dashboard/AccountBanner.js

```diff
@@ -185,28 +184,61 @@ const BannerWithProbability = ({ activeBalance }) => {
         align-items: center;
       `}
     >
-      <span css="margin-right: 8px">
-        <span
-          css={`
-            color: ${theme.accent};
-          `}
-        >
-          {probabilityText} probability{' '}
-        </span>
-        to be drafted
-      </span>
-      <Help hint="How is the probability calculated?">
-        The probability of being drafted is calculated dividing your active ANJ
-        balance by the total active ANJ in the Court during the current term.
-      </Help>
+      {(() => {
+        if (probablilityTooLow) {
+          return (
+            <span css="margin-right: 8px">
+              Activate more ANJ to increase your chances of being selected as a
+              juror
+            </span>
+          )
+        }
+        return (
+          <>
+            <span css="margin-right: 8px">
+              On average, you will be drafted
+              <span
+                css={`
+                  color: ${theme.accent};
+                `}
+              >
+                {` 1 on every ${chances} times `}
+              </span>
+            </span>
+            <Help hint="How is the probability calculated?">
+              Probability of being drafted is calculated dividing your active
+              ANJ balance by the total Court active ANJ, during the current
+              term.
+            </Help>
+          </>
+        )
+      })()}
     </div>
   )
 
   const paragraph =
     'The more ANJ you activate, more chances you have to be drafted to arbitrate a dispute'
-  return (
+
+  // TODO - change this for the loading indicator once ready
+  return fetchingTotalBalance ? (
+    <LoadingRing />
```

Comment:
Could we wrap this in a div of `{6 * GU}px` height ? Otherwise the balances misalign when this loading is rendered